test/index.js

```diff
@@ -166,4 +166,19 @@ describe('onBuild()', () => {
 
     expect(await pathExists(`${PUBLISH_DIR}/subdir/dummy.txt`)).toBeTruthy()
   })
+
+  test.each([
+    { FUNCTIONS_SRC: 'functions', resolvedFunctions: 'functions' },
+    { FUNCTIONS_SRC: undefined, resolvedFunctions: 'netlify-automatic-functions' },
```

Comment:
The first case is when a site configured a Functions directory. The second is when a site did not.